torchrl/collectors/distributed/ray.py

```diff
@@ -577,17 +591,33 @@ def stop_remote_collectors(self):
             )  # This will interrupt any running tasks on the actor, causing them to fail immediately
 
     def iterator(self):
```

Comment:
Should iterator raise an error for the async case? Or what happens when trying to iterate over this collector when replaybuffer was passed?

Reply:
You should get None outputs, and the buffer is filled each time you call next()